my_enumerable.rb

```diff
@@ -0,0 +1,17 @@
+module MyEnumerable
+  def all?
+    each { |num| return false unless yield num }
+    true
+  end
```

Comment:
- Please fix this method `all` because now it doesn't work and always have in mind that This method returns true if the block never returns false or nil
`list = MyList.new(1, 2, 3, 4)`
`list.all? {|e| e < 5}`
`true`

If I do 
`list.any? {|e| e > 5}`
`false`

Reply:
Thank you for the deep explanation @DeliceLydia. 😃 🙏  
Yes, we checked it now. By coincident we deleted a piece of code after yield, by deleting a comment that was on this line.